precompiles/addr/addr.go

```diff
@@ -75,6 +91,8 @@ func (p PrecompileExecutor) Execute(ctx sdk.Context, method *abi.Method, _ commo
 		return p.getSeiAddr(ctx, method, args, value)
 	case GetEvmAddressMethod:
 		return p.getEvmAddr(ctx, method, args, value)
+	case AssociateWithGas:
+		return p.associateWithGas(ctx, method, args, value)
```

Comment:
this would be a quite expensive call so we'd want to modify the gas cost for it in `RequiredGas()` function

Reply:
Don't have a good gauge on this so I set it to 50000 for now, does that make sense?

Reply:
yeah that should be enough